jest.config.ts

```diff
@@ -64,114 +35,32 @@ const config: JestConfigWithTsJest = {
-     * coverage information will be collected for it even if no tests exist for this file and
-     * it's never required in the test suite.
-     */
     collectCoverageFrom: ['<rootDir>/projects/**/*.ts'],
-
-    /**
-     * An array of regexp pattern strings that are matched against
-     * all file paths before executing the test. If the file path matches
-     * any of the patterns, coverage information will be skipped.
-     */
     coveragePathIgnorePatterns: [
         'node_modules',
         'schematics',
         'load-assets.ts',
         '.spec.ts',
         '.cy.ts',
     ],
-
-    /**
-     * A map from regular expressions to module names that allow to stub out resources,
-     * like images or styles with a single module. Modules that are mapped to an alias are
-     * un mocked by default, regardless of whether auto mocking is enabled or not.
-     * Use <rootDir> string token to refer to rootDir value if you want to use file paths.
-     * Additionally, you can substitute captured regex groups using numbered back references.
-     */
     moduleNameMapper: pathsToModuleNameMapper(compilerOptions.paths, {
         prefix: `<rootDir>/${compilerOptions.baseUrl}/`
             .replaceAll('./', '/')
             .replaceAll(/\/\/+/g, '/'),
     }),
-
-    /**
-     * An array of regexp pattern strings that are matched against all module paths before those
-     * paths are to be considered 'visible' to the module loader. If a given module's path
-     * matches any of the patterns, it will not be require()-able in the test environment.
-     */
     modulePathIgnorePatterns: ['.cache', 'dist', '<rootDir>/dist/'],
-
-    /**
-     * A list of reporter names that Jest uses when writing coverage reports.
-     * Any istanbul reporter can be used.
-     */
     coverageReporters: ['lcov', 'clover'],
-
-    /**
-     * The directory where Jest should store its cached dependency information.
-     */
     cacheDirectory: '<rootDir>/node_modules/.cache/jest',
-    /**
-     * A number limiting the number of tests that are allowed to run at the same time when
-     * using test.concurrent. any test above this limit will be queued and executed once
-     * a slot is released.
-     */
     maxConcurrency: maxParallel,
-
-    /**
-     * Specifies the maximum number of workers the worker-pool will spawn for running tests.
-     * In single run mode, this defaults to the number of the cores available
-     * on your machine minus one for the main thread.
-     */
     maxWorkers: maxParallel,
-
-    /**
-     * Display individual test results with the test suite hierarchy.
-     */
-    verbose: true,
-
-    /**
-     * By default, Jest runs all tests and produces all errors into the console upon completion.
-     * The bail config option can be used here to have Jest stop running tests after n failures.
-     * Setting bail to true is the same as setting bail to 1
-     */
+    verbose: silent,
```

Comment:
Shouldn't it be `!silent`?

Reply:
No ))
<img width="988" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/dcb971d8-2774-4d37-93ee-ccd4a938b68a">